python/GafferUI/ExpressionUI.py

```diff
@@ -174,6 +175,46 @@ def __init__( self, node, **kw ) :
 
 		self.__update()
 
+	def node( self ) :
+
+		return self.__node
+
+	def textWidget( self ) :
+
+		return self.__textWidget
+
+	__popupMenuSignal = Gaffer.Signal2()
+	## This signal is emitted whenever a popup menu for a plug is about
+	# to be shown. This provides an opportunity to customise the menu from
+	# external code. The signature for slots is ( menuDefinition, widget ),
+	# and slots should just modify the menu definition in place.
+	@classmethod
+	def popupMenuSignal( cls ) :
```

Comment:
We're not consistent on this throughout Gaffer, but "context menu" seems to predominate over "popup menu" in our API terminology, and since all menus popup, also seems to be slightly more specific. But `contextMenuSignal()` is also a (non-static) method on the Widget base class, so how abouts we call this `expressionContextMenuSignal()`?